WordPress/Classes/ViewRelated/Reader/WPStyleGuide+Reader.swift

```diff
@@ -85,7 +85,7 @@ extension WPStyleGuide {
     }
 
     public class func readerCrossPostSubtitleAttributes() -> [String: AnyObject] {
-        let font = WPStyleGuide.notoBoldFontForTextStyle(Cards.crossPostSubtitleTextStyle)
+        let font = WPStyleGuide.notoFontForTextStyle(Cards.crossPostSubtitleTextStyle)
```

Comment:
This is my bad, but previously we had this here:

```
public class func readerCrossPostBoldSubtitleAttributes() -> [String: AnyObject] {
   let font = WPStyleGuide.notoBoldFontForTextStyle(Cards.titleTextStyle)
   let font = WPFontManager.systemBoldFont(ofSize: fontSize)
```

We still want to use the system font for the subtitle.